### ISSUE TITLE
Change colors of charts to match contrast ratio standards

### ISSUE THREAD

**Issue by papajohn** [data-8/datascience#640]
The "gold" color is too light, and gray background is too dark, on all datascience slides. Let's update that with a bright blue like 2E51FF and a dark gold like 706700.

Ideally this will be done by noon on Wed 5/28 so that I can use it to make slides that afternoon. (But if it's going to take longer, please let me know.)

**Comment by pancakereport** [comment]
looking at this now. hopefully done by 5pm

**Comment by pancakereport** [comment]
Current colors for graphs like the [histograms here](https://github.com/data-8/textbook/blob/main/chapters/12/1/AB_Testing.ipynb) are yellow #fad366 and blue #475c76 with a gray background #f0f0f0. yellow fails all tests according to [this color contrast checker](https://webaim.org/resources/contrastchecker/) while blue seems to be fine for the "graphical objects and user interface components."

i'll be updating only the yellow to @papajohn 's suggested #706700 which passes the same as #475c76. background and dark blue color will remain the same.

**Comment by papajohn** [comment]
Sean did this already in a branch.

**Comment by papajohn** [comment]
https://github.com/sean-morris/datascience

**Comment by pancakereport** [comment]
@sean-morris if you want to PR i'll link this issue and approve so we can close the loop on this

**Comment by sean-morris** [comment]
Ok cool will do tonight
